### PR TITLE
feat: generic button

### PR DESCRIPTION
**Pull request by SebasthianSalpozz**

### CODE REVIEW

File: apps/todo-list-client/src/app/generic-button/generic-button.component.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<button type="button" class="m-2 rounded-xl bg-white p-4 text-center font-bold">
```
Comment (MorveN11) [inline apps/todo-list-client/src/app/generic-button/generic-button.component.html]: add de event to the button please, like (event)='changeAction()'

File: apps/todo-list-client/src/app/generic-button/generic-button.component.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-generic-button',
+  templateUrl: './generic-button.component.html',
+})
+export abstract class InputButtonUnitComponent {
```
Comment (MorveN11) [inline apps/todo-list-client/src/app/generic-button/generic-button.component.ts]: the selector names app-generic-button and your class names InputButtonUnitComponent, please change the name to a appropriate name like GenericButtonComponent

File: apps/todo-list-client/src/app/generic-button/generic-button.component.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  abstract title: string;
+
+  abstract changeAction(): void;
```
Comment (MorveN11) [inline apps/todo-list-client/src/app/generic-button/generic-button.component.ts]: Maybe a better name is onClick for this method, please consider that

**Review by MorveN11 — COMMENTED** [review]
Please @SebasthianSalpozz review all the comments

File: apps/todo-list-client/src/app/generic-button/generic-button.component.html (hunk 1 above)
Comment (SebasthianSalpozz) [reply apps/todo-list-client/src/app/generic-button/generic-button.component.html]: I'm done with this, thank you

File: apps/todo-list-client/src/app/generic-button/generic-button.component.ts (hunk 3 above)
Comment (SebasthianSalpozz) [reply apps/todo-list-client/src/app/generic-button/generic-button.component.ts]: I'm done with this, thank you

**Review by MorveN11 — APPROVED** [review]
Good Job @SebasthianSalpozz

**Pull request merged**